blueflood-core/src/test/java/com/rackspacecloud/blueflood/utils/InMemoryMetadataIO.java

```diff
@@ -6,13 +6,14 @@
 import com.google.common.collect.Table;
 import com.google.common.collect.Tables;
 import com.rackspacecloud.blueflood.io.MetadataIO;
+import com.rackspacecloud.blueflood.io.TestIO;
 import com.rackspacecloud.blueflood.types.Locator;
 
 import java.io.IOException;
 import java.util.Map;
 import java.util.Set;
 
-public class InMemoryMetadataIO implements MetadataIO {
+public class InMemoryMetadataIO implements MetadataIO, TestIO {
```

Comment:
It's just an overachiever class, it implements multiple interfaces. :-) I am also creating something like that for the Enum stuff. It's perfectly ok, I think.